org/wildfly/undertow/listener/capability.adoc

```diff
@@ -0,0 +1,30 @@
+NAME: org.wildfly.undertow.listener
+
+DESCRIPTION: Provides an Undertow listener
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: false
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.wildfly.extension.undertow.ListenerService<T>
```

Comment:
In this case T is a self parameter (depending on the type of listener), but I don't really want ListenerService to be a public API. A new interface needs to be extracted from this to represent the actual contract exposed by the service.